BrainPortal/app/views/data_providers/_one_data_provider_table.html.erb

```diff
@@ -76,15 +76,19 @@
   %>
 
   <% t.column("Alive?", :alive) do |dp| %>
-    <% if dp.online? %>
-      <%= staggered_loading "span", url_for(:action  => :is_alive, :id  => dp.id), :error_message  => "<span class=\"warning\">???</span>" do %>
-        <span class="loading_message"><%= html_colorize("(...)","magenta") %></span>
+    <%= on_click_ajax_replace( { :element  => "div",
```

Comment:
It's still doing a full round-trip to the server side. The original page already filled all the rows for offline data providers with `No` in the "Is Alive" column and didn't even check them when doing its staggered loading.